Automattic/pocket-casts-android
# End of Year: Show top 1 podcast and top 5 podcasts

| 📘 Project: #410  |
|:---:|

iOS PR: https://github.com/Automattic/pocket-casts-ios/pull/415

## Description
Adds the story for the top 1 podcast and the top 5 podcasts. 

PS: Ignore hard-coded strings and placeholder layouts/ design in StoryViews.

## To test

## Testing Instructions
1. Set `END_OF_YEAR_ENABLED` feature flag to `true` in `base.gradle`
2. Run the app and make sure you're logged in to an account with a few items on Listening History
3. When the prompt appears, tap "View My 2022"
4. Skip to the 5th story
5. ✅ Check that it shows your most listened podcast, together with the number of episodes and listening time
6. Skip to the 6th story
7. ✅ Check that it shows your top 5 podcasts

## Screenshots or Screencast 
| Top 1 podcast | Top 5 podcasts |
| -------------- | --------------- |
<img width=320 src="https://user-images.githubusercontent.com/1405144/199691888-aa727335-42bc-4e84-a110-9a417720a945.png"/> | <img width=320 src="https://user-images.githubusercontent.com/1405144/199690306-d9fc34a1-57a8-4e84-a2c3-c7a114b956fe.png"/>


## Checklist
N/A (added placeholder layouts + strings)

 
#### I have tested any UI changes...
